cmd/ocm/create/cluster/cmd.go

```diff
@@ -324,6 +328,15 @@ func init() {
 	)
 	arguments.SetQuestion(fs, "subscription-type", "Subscription type:")
 	Cmd.RegisterFlagCompletionFunc("subscription-type", arguments.MakeCompleteFunc(getSubscriptionTypeOptions))
+
+	fs.BoolVar(
+		&args.marketplaceGcpTerms,
+		"marketplace-gcp-terms",
+		false,
```

Comment:
should the default value be true, instead?

Reply:
By default we are setting it to false so that the user can override it to true when the terms and conditions are accepted. Otherwise they can skip this without noticing if it is set to true

Reply:
yes, that makes sense to me. Thanks for clarifying.